src/views/components/edit-model-form.jsx

```diff
@@ -126,6 +145,10 @@ export default class EditModelForm extends React.Component {
                     Faulty? <input type="checkbox" value={this.state.isFaulty} onChange={this.changeIsFaulty.bind(this)} /><br/>
                     Fault Description:<br/>
                     <textarea rows="4" cols="50"  value={this.state.faultDescription} onChange={this.changeFaultDescription.bind(this)} placeholder='Description' /><br/><br/>
+                    <p>Model photo thumbnail:</p>
+                    <input type='file' accept='image/*' onchange={this.changePhoto.bind(this)} capture />
+                    <img id='thumbnail-preview' />
```

Comment:
wouldn't `src = this.state.photo` work? Which would mean lines 97 and 98 aren't needed